packages/logging/README.md

```diff
@@ -1,6 +1,10 @@
 # @google-cloud/logging ([Beta][versioning])
 > Google Stackdriver Logging Client Library for Node.js
 
+This module allows you to work with the Stackdriver Logging API. If you are already using winston
```

Comment:
I would link `winston` to https://github.com/winstonjs/winston and `Bunyan` (capitalized) to https://github.com/trentm/node-bunyan.